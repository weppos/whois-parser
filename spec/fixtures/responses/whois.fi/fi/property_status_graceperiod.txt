
domain.............: kalatalousministeri.fi
status.............: Grace Period
created............: 14.3.2018 17:47:42
expires............: 14.3.2020 17:47:42
available..........: 14.4.2020 17:47:42
modified...........: 6.11.2019
RegistryLock.......: no

Nameservers

nserver............: ns-184-c.gandi.net [OK]
nserver............: ns-196-a.gandi.net [OK]
nserver............: ns-73-b.gandi.net [OK]

DNSSEC

dnssec.............: no

Holder

name...............: Digiteam Oy
register number....: 2823344-7
address............: Myllyniityntie 71
address............: 08100
address............: Lohja
country............: Finland
phone..............: +358.503716096
holder email.......: 

Registrar

registrar..........: Gandi SAS
www................: www.gandi.net

Tech

name...............: Seravo Oy
email..............: hostmaster@seravo.com

>>> Last update of WHOIS database: 22.3.2020 13:47:27 (EET) <<<


Copyright (c) Finnish Transport and Communications Agency Traficom

