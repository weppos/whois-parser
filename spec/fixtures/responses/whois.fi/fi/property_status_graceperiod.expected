#disclaimer
  %s == "Copyright (c) Finnish Transport and Communications Agency Traficom"


#domain
  %s == "kalatalousministeri.fi"

#domain_id
  %s %ERROR{AttributeNotSupported}


#status
  %s == :registered

#available?
  %s == false

#registered?
  %s == true


#created_on
  %s %CLASS{time}
  %s %TIME{2018-03-14 17:47:42}

#updated_on
  %s %CLASS{time}
  %s %TIME{2019-11-06}

#expires_on
  %s %CLASS{time}
  %s %TIME{2020-03-14 17:47:42}


#registrar
  %s.name == "Gandi SAS"
  %s.url  == "www.gandi.net"

#registrant_contacts
  %s %ERROR{AttributeNotSupported}

#admin_contacts
  %s %CLASS{array}
  %s %SIZE{1}
  %s[0] %CLASS{contact}
  %s[0].type          == Whois::Parser::Contact::TYPE_ADMINISTRATIVE
  %s[0].id            == "2823344-7"
  %s[0].name          == "Digiteam Oy"
  %s[0].address       == "Myllyniityntie 71"
  %s[0].city          == "Lohja"
  %s[0].zip           == "08100"
  %s[0].state         == nil
  %s[0].country       == "Finland"
  %s[0].country_code  == nil
  %s[0].phone         == "+358.503716096"
  %s[0].fax           == nil
  %s[0].email         == ""
  %s[0].created_on    == nil
  %s[0].updated_on    == nil

#technical_contacts
  %s %CLASS{array}
  %s %SIZE{1}
  %s[0] %CLASS{contact}
  %s[0].type          == Whois::Parser::Contact::TYPE_TECHNICAL
  %s[0].id            == nil
  %s[0].name          == "Seravo Oy"
  %s[0].address       == nil
  %s[0].city          == nil
  %s[0].zip           == nil
  %s[0].state         == nil
  %s[0].country       == nil
  %s[0].country_code  == nil
  %s[0].phone         == nil
  %s[0].fax           == nil
  %s[0].email         == "hostmaster@seravo.com"
  %s[0].created_on    == nil
  %s[0].updated_on    == nil


#nameservers
  %s %CLASS{array}
  %s %SIZE{3}
  %s[0] %CLASS{nameserver}
  %s[0].name == "ns-184-c.gandi.net"
  %s[1] %CLASS{nameserver}
  %s[1].name == "ns-196-a.gandi.net"
  %s[2] %CLASS{nameserver}
  %s[2].name == "ns-73-b.gandi.net"
